frontend/src/components/inputs/Selector.vue

```diff
@@ -43,6 +43,15 @@
           <span>{{ option.label }}</span>
         </div>
       </div>
+      <div class="items-end">
+        <div
+          class="arrow-back text-[2.2rem] after:content-['\1F870'] cursor-pointer text-sea-green"
```

Comment:
Same issue as Daniel has. The character is not rendering as an arrow. For me both on Chrome and Firefox.

Reply:
Waiting for @olympe-silve to provide me with an svg so i can proceed with solving this problem.

Reply:
@GabrielBuragev thank you for the reminder. here it is https://drive.google.com/file/d/1P9002z_a2jcFToCRSH6IG2KAFKt5JCNj/view?usp=sharing